hs/Scripts.hs

```diff
@@ -89,8 +90,14 @@ postScripts = sequence_ (map toScript [
                                           "/static/js/graph/objects/node.js",
                                           "/static/js/post/fill_textboxes.js",
                                           "/static/js/graph/create_data.js",
-                                          "/static/js/post/update_categories.js"])
-
+                                          "/static/js/post/update_categories.js",
+                                          "static/js/common/course_description.js",
```

Comment:
@Ian-Stewart-Binks That's fine, your comment is still valid. @christinem please add the `/`.